_i18n/es.yml

```diff
@@ -87,385 +87,385 @@ home:
   moneropedia_para: ¿Quieres ver las definiciones de términos y conceptos usados en Monero? Aquí encontrarás una lista alfabética de términos de los proyectos Monero y Kovri y sus significados.
   moneropedia_button: Leer Moneropedia
   user_guides: Guías de usuario
-  user_guides_para: Guías que describen varios procesos paso a paso, separadas por categoría. Cubren todo desde cómo crear una billetera, a cómo editar este mismo sitio web.
+  user_guides_para: Guías que describen varios procesos paso a paso, separadas por categoría. Cubren todo desde cómo crear un monedero, hasta cómo editar este mismo sitio web.
   user_guides_button: Ver guías de usuario
   faq: FAQ
   faq_para: Hemos escuchado muchas preguntas a lo largo de los años y hemos compilado un FAQ variado y detallado para tu conveniencia. No te preocupes, si tus preguntas no están aquí, puedes hacerlas por los canales de comunicación de la comunidad.

```

Comment:
I think this is not your translation, but it is too long. Could you rephrase it?

Reply:
I've already updated the file with that upgrade and some other corrections. It's ready to be tested again if you can, thanks for your feedback @el00ruobuob

Reply:
i can't see any change here. Did you push your changes?

Reply:
Ok, i understand. You copied your changes on your master branch.

Reply:
Yes, on my forked version, I was not sure where to make those updates so I made them there. Can you see it now?

Reply:
No...

Reply:
The easiest here, i believe:
* Save your latest file in another place on your pc
* Remove your local clone (whole directory), then
* `git clone -b BlackLotus64-patch-1 https://github.com/BlackLotus64/monero-site`
* copy back you file
* `git commit --amend`
* `git push --force`

Reply:
@erciccione could you help @BlackLotus64 here? I've not that much experience on git...

Reply:
Sure. @BlackLotus64 please ping me on `#monero-translations` when you are online, so i can help you with these

Reply:
Hey @BlackLotus64, I would like to invite you to check the [Spanish Terminology Wiki](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/spanish-terminology) created in the [Taiga Monero Localization](https://taiga.getmonero.org/project/erciccione-monero-localization/) instance workgroup for the Spanish translations. You could support yourself and contribute in adding more words. You can also comment and make any changes if needed. There  is also a chat where you could meet more Spanish contributors, it's `#monero-es`. Awesome work :) :+1: